frontends/java/src/main/java/ossf/fuzz/introspector/soot/CallGraphGenerator.java

```diff
@@ -674,6 +647,67 @@ private String mergeClassName(Set<String> classNameSet) {
     return mergedClassName.toString();
   }
 
+  private Callsite handleStatement(Stmt stmt, String sourceFilePath) {
```

Comment:
Could you make the function names more descriptive and some descriptive comments/function-level doc? `handle` is too generic since it does not say much about the semantics of the function.

Reply:
Changed